internal/resources/policy/kind/network/recipe/allow_all_to_pods_flatten_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package recipe
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	policyrecipenetworkmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/network"
+	policyrecipenetworkcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/network/common"
+)
+
+func TestFlattenAllowAllToPods(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		description string
+		input       *policyrecipenetworkmodel.V1alpha1CommonPolicySpecNetworkV1AllowAllToPods
+		expected    []interface{}
+	}{
+		{
+			description: "check for nil allow-all-to-pods recipe network policy",
+			input:       nil,
+			expected:    nil,
+		},
+		{
+			description: "normal scenario with valid allow-all-to-pods recipe network policy",
```

Comment:
Are you covering this line 
if allowAllToPods.ToPodLabels != nil {

in the unti tests ?

Reply:
<img width="1308" alt="Screenshot 2023-06-06 at 11 05 20 AM" src="https://github.com/vmware/terraform-provider-tanzu-mission-control/assets/32017029/8b08ed32-096e-4da0-8e7e-bdedf0344fc5">